desktop/app-handlers/printer/index.js

```diff
@@ -18,11 +16,17 @@ module.exports = function() {
 
 		log.debug( 'Printing HTML' );
 
-		printer.loadURL( 'data:text/html,' + encodeURIComponent( html ) );
+		const file = 'data:text/html;charset=UTF-8,' + encodeURIComponent( html );
+		printer.loadURL( file );
 
 		printer.webContents.on( 'dom-ready', function() {
+			const options = { silent: false }; // ask the user for print settings
```

Comment:
Note: this is the default but just wanted to make it explicit. Also, specifying the options explicitly allows us to use the `webContents.print` callback to log error messages.